src/configui/useConfig.tsx

```diff
@@ -60,7 +60,7 @@ export default function (editedPath: string, inputConfig: Config) {
     .map(([prop, val]) =>
       prop === "tags" && val.length === 1 ? ["tag", val[0]] : [prop, val]
     )
-    .filter(([_, val]) => val !== "" && val !== null && val !== []);
+    .filter(([_, val]) => val !== "" && val !== null);
```

Comment:
Fixes the following build error:
```
ERROR in /home/user/Documents/plugin-kanban--fork/src/configui/useConfig.tsx
63:56-66
[tsl] ERROR in /home/user/Documents/plugin-kanban--fork/src/configui/useConfig.tsx(63,57)
      TS2839: This condition will always return 'true' since JavaScript compares objects by reference, not value.
```